packages/commonwell-sdk/src/common/make-jwt.ts

```diff
@@ -1,5 +1,5 @@
 import * as jwt from "jsonwebtoken";
-import { PurposeOfUse } from "../models/purpose-of-use";
+import { PurposeOfUse } from "@metriport/shared";
```

Comment:
I don't think we can do this. Shared AFAIK is not published and we don't want to publish it, so we can keep it streamlined the development without publishing packages, no?

Reply:
Oh, it's [published](https://www.npmjs.com/package/@metriport/shared)! 🤔

Reply:
I thought this would reference the local package as it was getting built, if not what would be the best approach to prevent duplicating this?